src/components/NavBar.module.css

```diff
@@ -0,0 +1,55 @@
+.root {
+  --icon-font: montserrat;
+  --font: robotoslab;
```

Comment:
- [ ] Kindly use the same fonts as shown in the [original design](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378). 😄 🙏 

>Please apply these changes to all of your elements accordingly!